sklearn/neighbors/setup.py

```diff
@@ -31,6 +31,11 @@ def configuration(parent_package='', top_path=None):
                          sources=['typedefs.pyx'],
                          include_dirs=[numpy.get_include()],
                          libraries=libraries)
+    config.add_extension("quad_tree",
+                         sources=["quad_tree.pyx"],
+                         include_dirs=[numpy.get_include()],
+                         libraries=libraries,
+                         extra_compile_args=["-O3"])
```

Comment:
It was actually a copy past from an other `setup.py`. I do not know enough to really explain but it appears to trigger some optimization level and it works on multiple compiler such as gcc and clang.  
I don't see a reason to keep it (as it was added pretty randomly) except if someone knowledgeable has an opinion on that.